src/file_lines.rs

```diff
@@ -0,0 +1,221 @@
+//! This module contains types and functions to support formatting specific line ranges.
+use std::{cmp, iter, str};
+
+use itertools::Itertools;
+use multimap::MultiMap;
+use rustc_serialize::{self, json};
+
+use codemap::LineRange;
+
+/// A range that is inclusive of both ends.
```

Comment:
Here and in LineRange, why is it inclusive both ends rather than inclusive below and exclusive above - that seems a more common convention.